### DISCUSSION TITLE
vx.exe not working for me

### DISCUSSION THREAD

**Discussion by le-rou** [VentuzTechnology/Ventuz.Extension.prerelease#8] [Q&A]
I can't get the vx.exe to work, it will always tell me the assembly was compiled for another processor. Obviously, my assembly was built for x64 or it could not run in Ventuz.

**Comment by VentuzTechnology** [comment]
Björn found the issue. The Release configuration was wrong.
[see Changelog](https://github.com/VentuzTechnology/Ventuz.Extension.prerelease/blob/main/VersionChangeLog.md)

currently uploading a new version

-Daniel

**Comment by VentuzDanielWiller** [comment]
Converted to [Issue 21](https://github.com/VentuzTechnology/Ventuz.Extension.prerelease/issues/21)
